packages/charts/src/vaadin-chart.js

```diff
@@ -428,7 +432,6 @@ class Chart extends ResizeMixin(ElementMixin(ThemableMixin(PolymerElement))) {
        */
       emptyText: {
         type: String,
-        value: ' ',
```

Comment:
**question:** Would it make sense to mark the PR as `fix:!` because of this change?

Reply:
In practical terms, nothing should change. With the previous default value, the chart would not show a no data message. With removing the default value, and [setting up a global default value of an empty string](https://github.com/vaadin/web-components/pull/3677/files#diff-672c1c097ba20dec3868a1ab9a23eab624d5a012a7e00a31c8c89d4a86e25b32R67), the chart also does not show a no data message.

Reply:
I also think it's unlikely that someone will have a use-case around this.